MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+    } 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Setting title for time editor
+
+You can set title to the time editor by handling the [Title]`Link`property in [DataFormTimeItem]`Link`. The default value of this property is `Choose Time`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "BirthTime")
+        {
+            (e.DataFormItem as DataFormTimeItem).Title = "Select time of birth";
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## CheckBox editor
+
+In `CheckBox` editor, the [SfCheckBox]`Link` control is loaded. By default, for `bool` data type property, the `CheckBox` editor will be loaded in data form.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+**Visual states of SfCheckBox**
+
+SfCheckBox support three visual states.
+  - Checked
+  - UnChecked
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [Display(Name = "Is Billable")]
+    public bool IsBillable { get; set; } = true;
+
+    [Display(Name = "Registered Member")]
+    public bool RegisteredMember { get; set; } = false;
+
+    [Display( Name = "Is Refundable")]
+    public bool? IsRefundable { get; set; } = null;
+
+{% endhighlight %}
+{% endtabs %}
+		
+## Switch Editor
+
+In switch editor, [Switch]`Link` is loaded, and DataForm `Switch` editor supports bool data type property.
+
+To add `Switch` editor in DataForm, register the editor as `Switch` for the required property using the [RegisterEditor]`Link` method.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+       dataForm.RegisterEditor("CellularData", DataFormEditorType.Switch);
+       dataForm.RegisterEditor("AirplaneMode", DataFormEditorType.Switch);
+
+        [Display(Name ="Cellular Data")]
+        public bool CellularData { get; set; } = true;
+
+        [Display(Name = "Airplane Mode")]
+        public bool AirplaneMode { get; set; }
+ 
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+## Combo Box editor
+
+In the Combo Box editor, the [SfComboBox]`Link` will be loaded.
+
+### Customizing ItemsSource of SfComboBox
+
+By default, the `ItemsSource` for SfComboBox is auto-generated for enum types and collection type properties. For other types, you can set the `ItemsSource` by using the [IDataFormSourceProvider]`Link`.
+
+#### Using IDataFormSourceProvider
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    public string ItemName { get; set; }
+    
+    public class SourceProviderExt : IDataFormSourceProvider
+    {
+        public override IList GetSource(string sourceName)
```

Comment:
public override **object** GetSource(string sourceName)

Test code before using in UG

Reply:
IList to override object changed